### DISCUSSION TITLE
VM Isolation

### DISCUSSION THREAD

**Discussion by thecyberduck** [utmapp/UTM#4323] [UTM Help (macOS)]
I want to use a VM on my Mac for testing. That’s why I wonder if I can use a VM as a isolated System? I know under windows that is possible but is UTM supports that too?

**Comment by ktprograms** [comment]
If you mean network isolated, then enable the "Isolate Guest from Host" checkbox.

**Reply by thecyberduck** [reply]
That’s one point. Another is, that noting from the VM can break out on my Mac. Like clipboard and hard drive.

**Reply by ktprograms** [reply]
For clipboard, you can disable clipboard sharing.

You can disable file sharing.
